installer/engine_installer/engine_installer.go

```diff
@@ -14,8 +14,172 @@
 
 package main
 
-import "fmt"
+import (
+	"html/template"
+	"strings"
 
-func main() {
-	fmt.Println("hello world")
+	"context"
```

Comment:
gofmt: imports should be ordered correctly and standard pkgs should be in the same group.